src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.html

```diff
@@ -1,84 +1,181 @@
+          <ng-container
+            [ngTemplateOutlet]="navItemTemplate"
+            [ngTemplateOutletContext]="{
+              disabled: !canRoute('materials'),
+              link: 'materials',
+              text: 'Materials'
+            }"
+          ></ng-container>
         </li>
       </ul>
       <div class="builder-navbar-wrapper__saving-indicator">
-        <span *ngIf="isSaving" class="builder-navbar-wrapper__saving-indicator">Saving changes . . . </span>
-        <span *ngIf="!isSaving" class="builder-navbar-wrapper__saving-indicator">All changes saved</span>
+        <span *ngIf="isSaving" class="builder-navbar-wrapper__saving-indicator"
+          >Saving changes . . .
+        </span>
+        <span *ngIf="!isSaving" class="builder-navbar-wrapper__saving-indicator"
+          >All changes saved</span
+        >
       </div>
     </div>
-
   </div>
 </div>
 
-<clark-popup *ngIf="showSubmission" (closed)="showSubmission = false">
-  <div #popupInner>
-    <clark-collection-selector [currentCollection]="store.learningObjectEvent.getValue().collection" (choice)="submitForReview($event)"
-      (cancel)="showSubmission = false"></clark-collection-selector>
-  </div>
-</clark-popup>
+<clark-submit *ngIf="showSubmission" [learningObject]="store.learningObject" [visible]="true" (close)="showSubmission = false; getCollectionName()"></clark-submit>
 
 <clark-context-menu (id)="submissionOptionsMenu = $event" (close)="toggleSubmissionOptionsMenu($event)">
   <div #contextMenu>
     <ul>
-      <li class="bad" (click)="cancelSubmission(); toggleSubmissionOptionsMenu($event)">
+      <li
+        class="bad"
+        (click)="cancelSubmission(); toggleSubmissionOptionsMenu($event)"
+      >
         <i class="far fa-ban"></i> Cancel Submission
       </li>
     </ul>
   </div>
 </clark-context-menu>
 
-<ng-template #navItemTemplate let-link="link" let-disabled="disabled" let-text="text">
-  <a *ngIf="!disabled" [ngClass]="{'new': isNewRoute(link)}" routerLink="{{ link }}" routerLinkActive="active" (click)="triggerRouteClick(link)">{{
-    text }}</a>
-  <a *ngIf="disabled" class="disabled" tip="Please give this learning object a name before adding outcomes or uploading materials"
-    tipLocation="above"><i class="fas fa-lock"></i>{{ text }}</a>
+<ng-template
+  #navItemTemplate
+  let-link="link"
+  let-disabled="disabled"
+  let-text="text"
+>
+  <a
+    *ngIf="!disabled"
+    [ngClass]="{ new: isNewRoute(link) }"
+    routerLink="{{ link }}"
+    routerLinkActive="active"
+    (click)="triggerRouteClick(link)"
+    >{{ text }}</a
+  >
+  <a
+    *ngIf="disabled"
+    class="disabled"
+    tip="Please give this learning object a name before adding outcomes or uploading materials"
+    tipLocation="above"
+    ><i class="fas fa-lock"></i>{{ text }}</a
+  >
+</ng-template>
+
+<ng-template #genericCollectionLogoTemplate>
```

Comment:
Most of the changes in this file are due to formatting. This is an actual change.